msi/GitProduct.wxs

```diff
@@ -46,6 +48,12 @@
           <Directory Id='BinFolder' Name='bin' />
         </Directory>
       </Directory>
+
+      <Directory Id='PersonalFolder' />
```

Comment:
You speak in riddles, sir. Why do we need a working folder? And why is it called `PersonalFolder` when it sounds more like it is a temporary folder?